core/src/main/java/com/linecorp/armeria/common/logging/DefaultRequestLog.java

```diff
@@ -535,33 +547,33 @@ private void propagateRequestSideLog(RequestLogAccess child) {
         // Update the available properties always by adding a callback,
         // because the child's properties will never be available immediately.
         child.whenAvailable(RequestLogProperty.REQUEST_START_TIME)
-             .thenAccept(log -> startRequest(log.requestStartTimeNanos(), log.requestStartTimeMicros()));
+                .thenAccept(log -> startRequest(log.requestStartTimeNanos(), log.requestStartTimeMicros()));
```

Comment:
Did you set up your IDE as recommended? https://armeria.dev/community/developer-guide#setting-up-your-ide

Can you revert the indentation changes made in this file?

Reply:
I'm sorry. I'll fix it quickly.